helm/nginx-ingress-controller-app/templates/np.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: {{ include "resource.default.name" . }}
   namespace: {{ .Release.Namespace }}
   labels:
+    app: nginx-ingress-controller
```

Comment:
FYI https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/helm/nginx-ingress-controller-app/templates/_helpers.tpl#L19-L20

Reply:
I see those labels on most nginx-ingress-controller apps, but not when I install it through Happa. Is this expected or am I doing something stupid?